packages/ssz/src/viewDU/listComposite.ts

```diff
@@ -66,6 +66,47 @@ export class ListCompositeTreeViewDU<
     return this.type.getViewDU(newRootNode) as this;
   }
 
+  /**
+   * Returns a new ListCompositeTreeViewDU instance with the values from `index` to the end of list
+   *
+   * ```ts
+   * const nodes = getChunkNodes()
+   * return listFromChunkNodes(node.clice(index))
+   * ```
+   *
+   * Note: If index === n, returns an empty list of length 0
+   *
+   */
+  sliceFrom(index: number): this {
+    // Commit before getting rootNode to ensure all pending data is in the rootNode
+    this.commit();
+
+    // If slicing from 0, no slicing is necesary
+    if (index <= 0) {
```

Comment:
from a library perspective, it might be nice to handle negative indices as starting from the end, similarly to how `Array#slice` now works.

```js
[1, 2, 3, 4, 5].slice(-1) // -> [5]
```

should be as simple as adding this above L85:
```ts
if (index < 0) {
  index = length + index
}
```